app/scripts/controllers/preferences.js

```diff
@@ -175,7 +176,13 @@ export default class PreferencesController {
     const suggested = this.getSuggestedTokens()
     const { rawAddress, symbol, decimals, image } = tokenOpts
     const address = normalizeAddress(rawAddress)
-    const newEntry = { address, symbol, decimals, image }
+    const newEntry = {
+      address,
+      symbol,
+      decimals,
+      image,
+      unlisted: contractMap[address] === undefined,
```

Comment:
Oh, the same normalization problem applies here as well

Reply:
Oops. https://github.com/MetaMask/metamask-extension/pull/9810/commits/a0fe3e9e9efec29cceaa8897096d3d7c6ad04b02 i moved the lowercase map to a new constants file and expanded the fix from the previous commit to this controller as well.

Reply:
Great! A shared constant seems like a good idea.